composer.json

```diff
@@ -24,7 +24,7 @@
         "guzzle/guzzle": "3.*",
         "imagine/imagine": "~0.6.1",
         "massive/search-bundle": "0.16.*",
-        "sulu/document-manager": "0.8.*",
+        "sulu/document-manager": "^0.8.2",
```

Comment:
Hm, I think I would go for `~0.8.2`, because that doesn't include `0.9.0`. I think the library is not stable enough to rely on the semantic versioning operator, and it would be closer to what it was before.